### DISCUSSION TITLE
List with extisting/known device cluster and device types

### DISCUSSION THREAD

**Discussion by Tropaion** [zigpy/zha-device-handlers#1501] [General]
I don't know if it already exists, but a comprehensive list with supported/known clusters and device types would be nice.
Would making a quirk, especially for newcomers way more easy.

**Comment by dmulcahey** [comment]
What do you mean? https://zigbeealliance.org/wp-content/uploads/2019/12/07-5123-06-zigbee-cluster-library-specification.pdf

**Reply by MattWestb** [reply]
The "new" ZCL R8 i have putting in the reference collection https://github.com/zigpy/zigpy/discussions/595#discussioncomment-1656808.

Profiles used in Zigpy https://github.com/zigpy/zigpy/tree/dev/zigpy/profiles.
Cluster used in Zigpy https://github.com/zigpy/zigpy/tree/dev/zigpy/zcl/clusters.

**Reply by Tropaion** [reply]
I know these, but that wasn't what I meant.
I also use the specification but it doesn't include the manufacturere specific clusters. For them, I always have to search for a while to get some information.

@MattWestb Thanks for the links, I was always searching the profiles to find similar ones in other quirks, didn't know this list.
Would be nice if this link collection would be in the #Readme

**Reply by MattWestb** [reply]
Some "collected" manufacture extended cluster with attributes and commands (most tuya) https://github.com/zigpy/zigpy/discussions/823.
I dont have the knowledge of Xiaomi / Aqara but they have there own standard of cluster and commands (as tuya have doing in later time) but i think some can being red in the Xiaomi quirks how its working.

**Reply by MattWestb** [reply]
I cant changing the Red.Me but have trying fixing little in the wiki and doing discussion that can being linked from the Wiki.
The maintainers must saying how they like to using the information and putting it the right place.

**Reply by Tropaion** [reply]
Which wiki? It shows me an empty page.

**Reply by MattWestb** [reply]
I using Zigpy for more general things that  is not radio lib related https://github.com/zigpy/zigpy/wiki but  its not so much for devs there. You can adding more if you is having more knowledge.

**Reply by dmulcahey** [reply]
If someone wants to PR wiki pages to the device handler repo for that it would be welcomed
